workers/default/xml/tasks.py

```diff
@@ -47,7 +47,6 @@ def process_object(self, obj):
             additional_params = self.get_param(self.get_param('params'))
         except KeyError:
             additional_params = None
-
```

Comment:
Warum wurde die Leerzeile entfernt? Ist jetzt nichts schlimmes, aber das sind so Dinge, weshalb man immer genau hinschauen muss was in einem Commit enthalten ist.